samples/openapi3/client/petstore/dart2/petstore_client_lib_fake/lib/auth/http_basic_auth.dart

```diff
@@ -11,12 +11,14 @@
 part of openapi.api;
 
 class HttpBasicAuth implements Authentication {
+  HttpBasicAuth(this.username, this.password);
```

Comment:
The problem with the constructor is that it is used in the `ApiClient` constructor without arguments

Reply:
This used to be different actually, but copied from @agilob's pr. I can revert back to my version.

Reply:
:heavy_check_mark: